content/id/docs/concepts/scheduling/scheduler-perf-tuning.md

```diff
@@ -0,0 +1,160 @@
+serta 10% untuk klaster dengan 5000 Node.
+
+Artinya, kube-scheduler selalu menskor paling tidak 5% dari klaster kamu, terlepas dari
+seberapa besar klasternya, kecuali kamu secara eksplisit mengatur `percentageOfNodesToScore`
+menjadi lebih kecil dari 5.
+
+Jika kamu ingin penjadwal untuk memasukkan seluruh Node di dalam klaster ke dalam penskoran,
+maka atur `percentageOfNodesToScore` menjadi 100.
+
+## Contoh
+
+Contoh konfigurasi di bawah ini mengatur `percentageOfNodesToScore` menjadi 50%.
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1alpha1
+kind: KubeSchedulerConfiguration
+algorithmSource:
+  provider: DefaultProvider
+
+...
+
+percentageOfNodesToScore: 50
+```
+
+
+##  Menyetel percentageOfNodesToScore
+
+`percentageOfNodesToScore` merupakan angka 1 sampai 100 dengan
+nilai bawaan yang dihitung berdasarkan ukuran klaster. Di sini juga terdapat
+batas bawah yang telah ditetapkan, yaitu 50 Node.
+
+{{< note >}}Pada klaster dengan kurang dari 50 Node layak, penjadwal masih
+terus memeriksa seluruh Node karena Node-Node layak belum mencukupi supaya
+penjadwal dapat menghentikan proses pencarian lebih awal.
+
+Pada klaster kecil, jika kamu mengatur `percentageOfNodesToScore` dengan angka kecil,
+pengaturan ini hampir atau sama sekali tidak berpengaruh, karena alasan yang sama.
+
+Jika klaster kamu punya ratusan Node, gunakan angka bawaan untuk opsi konfigurasi ini.
+Mengubah angkanya kemungkinan besar tidak akan mengubah kinerja penjadwal secara berarti.
+{{< /note >}}
+
+Sebuah catatan penting yang perlu dipertimbangkan saat mengatur angka ini adalah
+ketika klaster dengan jumlah Node sedikit diperiksa untuk kelayakan, beberapa Node
+tidak dikirim untuk diskor bagi sebuah Pod. Hasilnya, sebuah Node yang mungkin memiliki
+nilai lebih tinggi untuk menjalankan Pod tersebut bisa saja tidak diteruskan ke fase penskoran.
+Hal ini berdampak pada penempatan Pod yang kurang ideal.
+
+Kamu harus menghindari pengaturan `percentageOfNodesToScore` menjadi sangat rendah,
+sehingga kube-scheduler tidak seringkali memberikan keputusan penempatan Pod yang tidak bagus.
+Hindari pengaturan persentase di bawah 10%, kecuali _throughput_ penjadwal sangat penting
+untuk aplikasi kamu dan skor dari Node tidak begitu penting. Dalam kata lain, kamu
+memilih untuk menjalankan Pod pada Node manapun selama Node layak.
+
+## Bagaimana Penjadwal Mengecek Node
+
+Bagian ini ditujukan untuk kamu yang ingin mengerti bagaimana fitur ini bekerja secara internal.
+
+Untuk memberikan semua Node di dalam klaster sebuah kesempatan yang adil untuk
+dipertimbangkan dalam menjalankan Pod-Pod, penjadwal mengecek Node satu persatu
+secara _round robin_. Kamu dapat membayangkan Node-Node ada di dalam sebuah array.
+Penjadwal mulai dari indeks array pertama dan mengecek kelayakan dari Node sampai
+jumlahnya telah mencukupi sesuai dengan `percentageOfNodesToScore`. Untuk Pod berikutnya,
+penjadwal meneruskan dari indeks array Node yang terhenti ketika memeriksa
```

Comment:
```suggestion
penjadwal melanjutkan dari indeks array Node yang terhenti ketika memeriksa
```

I feel like "melanjutkan" is more specific to "continue".
"Meneruskan" also has other equivalent, which is "forwarding".

Reply:
Fixed in https://github.com/kubernetes/website/pull/20260/commits/f9ef26b8d200c1872948e4dbb15cdbcc45ba7b0f